filter/store.go

```diff
@@ -159,10 +150,8 @@ func (store *Store) makeRequest(method, url string, body io.Reader) ([]byte, err
 	if err != nil {
 		return nil, err
 	}
-	request.Header.Set("Internal-Token", store.filterAPIAuthToken)
-	request.Header.Set("Authorization", store.serviceToken)
 
-	response, responseError := store.httpClient.Do(request)
+	response, responseError := store.httpClient.Do(context.Background(), request)
```

Comment:
It's odd that we use this function to make requests to the filter api, rather than using the go-ns filter client. Not expecting you to make the change, just an observation!